syntax = "proto3";
import "ClientPublic.proto";

message Chat_ChatReq
{
	string chat_msg = 1;
	int32 channel = 2;
}

message Chat_ChatReply
{
	int32 ret = 1;
	string chat_msg = 2;
}